src/BaseCommand.js

```diff
@@ -42,6 +42,7 @@ class BaseCommand extends Command {
     await super.init()
     // setup a prompt that outputs to stderr
     this.prompt = inquirer.createPromptModule({ output: process.stderr })
+    process.env.__OW_USER_AGENT = 'aio-cli-plugin-app@' + require('../package.json').version
```

Comment:
just a reviewer note: even though it's not documented in oclif, the command's `init` function is always called before a `run` is called: https://github.com/oclif/core/blob/68110ee2a4839f0b3ab299ae775d3af1f0c107bc/src/command.ts#L359

If we want to add more details like a browser user agent, we can get it from `.config.versionDetails()` (but this is not documented): https://github.com/oclif/core/blob/68110ee2a4839f0b3ab299ae775d3af1f0c107bc/src/config/config.ts#L187

Reply:
Done! Thanks